nilearn/connectome/tests/test_group_sparse_cov.py

```diff
@@ -35,55 +35,11 @@ def test_check_estimator_group_sparse_covariance_cv(estimator, check, name):  #
 @pytest.mark.parametrize(
     "estimator, check, name",
     check_estimator(
-        estimator=[GroupSparseCovarianceCV()],
+        estimator=[GroupSparseCovarianceCV(), GroupSparseCovariance()],
         valid=False,
         extra_valid_checks=extra_valid_checks,
     ),
 )
-def test_check_estimator_invalid_group_sparse_covariance_cv(
```

Comment:
GroupSparseCovariance and GroupSparseCovarianceCV now validated the same checks so we can use a single parametrized test for both